src/overlays/actors/ovl_player_actor/z_player.c

```diff
@@ -1777,53 +1780,53 @@ void func_808326F0(Player* this) {
     }
 }
 
-u16 func_8083275C(Player* this, u16 sfxId) {
+u16 Player_ApplyFloorSfxOffset(Player* this, u16 sfxId) {
     return sfxId + this->floorSfxOffset;
 }
 
-void func_80832770(Player* this, u16 sfxId) {
-    Player_PlaySfx(this, func_8083275C(this, sfxId));
+void Player_PlaySfxWithFloorOffset(Player* this, u16 sfxId) {
+    Player_PlaySfx(this, Player_ApplyFloorSfxOffset(this, sfxId));
 }
 
-u16 func_808327A4(Player* this, u16 sfxId) {
+u16 Player_ApplyFloorAndAgeSfxOffsets(Player* this, u16 sfxId) {
     return sfxId + this->floorSfxOffset + this->ageProperties->unk_94;
 }
 
-void func_808327C4(Player* this, u16 sfxId) {
-    Player_PlaySfx(this, func_808327A4(this, sfxId));
+void Player_PlaySfxWithFloorAndAgeOffsets(Player* this, u16 sfxId) {
+    Player_PlaySfx(this, Player_ApplyFloorAndAgeSfxOffsets(this, sfxId));
 }
 
-void func_808327F8(Player* this, f32 arg1) {
+void Player_PlayWalkingSfx(Player* this, f32 linkSpeed) {
```

Comment:
id be careful about calling this `linkSpeed`, cause this isnt necessarily his speed. It seems to be moreso just a pitch scaling factor that gets passed to the audio playback code.
We can wait for audio to document this side and name it after that, or go with an intermediate name like `pitchAdjustment` or something, idk

Also, thoughts on `Player_PlaySteppingSfx`? This way it covers the cases of both running and walking, depending on the "speed" or pitch value passed in

Reply:
I like `Player_PlaySteppingSfx`.
Imo it is actually link's speed as all uses (in which it's not a constant) pass `this->speedXZ` as it's value. Don't have a strong preference for the name tho, done.

Reply:
thanks.

It really isnt his speed though right, precisely because of the constant cases. Its just that in some cases `speedXZ` is also used directly to modify the pitch of the footstep sounds. It makes more sense to name if after what it does for the code its being used for, not just what the input is